[2022-01-14 20:53:58,287] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_04 scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-01-14 20:54:00,084] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_04 scheduled__2022-01-13T00:00:00+00:00 [queued]>
[2022-01-14 20:54:00,085] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-14 20:54:00,085] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-14 20:54:00,085] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-14 20:54:00,684] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_04> on 2022-01-13 00:00:00+00:00
[2022-01-14 20:54:00,697] {standard_task_runner.py:52} INFO - Started process 52934 to run task
[2022-01-14 20:54:01,401] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_04', 'scheduled__2022-01-13T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpooa4hfne', '--error-file', '/tmp/tmpsq27v7tz']
[2022-01-14 20:54:01,500] {standard_task_runner.py:77} INFO - Job 67: Subtask task_04
[2022-01-14 20:54:06,685] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_04 scheduled__2022-01-13T00:00:00+00:00 [running]> on host airflowvm
[2022-01-14 20:54:12,535] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_04
AIRFLOW_CTX_EXECUTION_DATE=2022-01-13T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-13T00:00:00+00:00
[2022-01-14 20:54:12,536] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-14 20:54:12,538] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-01-14 20:54:12,645] {subprocess.py:85} INFO - Output:
[2022-01-14 20:54:15,734] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-14 20:54:17,443] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_04, execution_date=20220113T000000, start_date=20220114T205358, end_date=20220114T205417
[2022-01-14 20:54:18,532] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-14 20:54:20,496] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
